<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diag_find2</class>
 <widget class="QWidget" name="diag_find2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>526</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column/>
     <column/>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_List" native="true">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QCommandLinkButton" name="commandLinkButton_Timestamps">
        <property name="text">
         <string>List &amp;timestamps</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCommandLinkButton" name="commandLinkButton_Branches">
        <property name="text">
         <string>List &amp;branches</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCommandLinkButton" name="commandLinkButton_Lyrics">
        <property name="text">
         <string>List l&amp;yrics</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Find command/text</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_List">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;List...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButton_Search">
        <property name="text">
         <string>&amp;Search</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit_command">
        <property name="placeholderText">
         <string>END();</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_FindNext">
        <property name="text">
         <string>Find ne&amp;xt</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBox_Starts">
        <property name="text">
         <string>&amp;Starts</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QWidget" name="widget_FindOptions" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="checkBox_Wrap">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>W&amp;rap</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_Whole">
           <property name="text">
            <string>&amp;Whole words</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_Backwards">
           <property name="text">
            <string>Back&amp;wards</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_Case">
           <property name="text">
            <string>&amp;Case-sensitive</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
